php/php-rr-sdk.md

```diff
@@ -186,41 +169,46 @@ use Temporal\Workflow;
 
 class UploadWorkflow
 {
-    public function run(Workflow\ContextInterface $ctx, string $input): string
+    public function run(string $input): string
     {
-        $promise = yield new Workflow\ExecuteActivy(
+        $promise = new Workflow\ExecuteActivy(
             'activityName',
             $input, 
-            100 // startToFinishTimeout, etc. 
+            [
+                ExecuteActivy::START_TO_FINISH_TIMEOUT   => 100,        
+                ExecuteActivy::SCHEDULE_TO_START_TIMEOUT => 10
+            ]       
         );        
     
+        // blocking operation to get the activity result
```

Comment:
It is kind of weird to have both get and yield. May be pass object type as a parameter to the ExecuteActivity and then just yield on the result? Something like:

```
$promise = new Workflow\ExecuteActivy(
            'activityName',
            $input,
            Obj::class, //result type
             [
                ExecuteActivy::START_TO_FINISH_TIMEOUT   => 100,        
                ExecuteActivy::SCHEDULE_TO_START_TIMEOUT => 10
              ]       
        ); 

  $result = yield $promise; // No get
  return $result; // not clear why getValue was needed
```

Reply:
I like this approach, we can always add more sugar later via activity builder